app/transport/rest/user.go

```diff
@@ -97,12 +122,37 @@ func (uh UserHandler) GetUserByID() http.Handler {
 		}
 
 		user := User{
-			ID:        entUser.ID,
-			FirstName: entUser.FirstName,
-			LastName:  entUser.LastName,
-			Email:     entUser.Email,
-			CreatedAt: entUser.CreatedAt,
+			ID:        solidUser.ID,
```

Comment:
The problem is that I have no idea what is solid user 😄

Reply:
I will get rid of this dude.  😄